manifest.json

```
@@ -0,0 +1,15 @@
+{
+  "addons": [
+    {
+      "id": "todotreeview",
+      "version": "1.0",
+      "mod_version": "3",
+      "description": "A TODO tree view for Lite XL",
```

```suggestion
      "description": "Todo tree viewer for annotations in code like `TODO`, `BUG`, `FIX`, `IMPROVEMENT`",
```
Same description used in the plugins repo.